### PR TITLE
Dawnpm plain language review fixes

### PR DESCRIPTION
**Pull request by dawnpm**
## PR Checklist
- [x] Test Federalist Preview link for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
- [x] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [x] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -215,37 +215,37 @@ get_started:
...
-          - name: A step-by-step indexing workflow
+          - name: A Step-by-step Indexing Workflow
             url: "/indexing/indexing-workflow.html"
       - name: Domain Level SEO Supports
         children:
-          - name: All about XML sitemaps
+          - name: All About XML Sitemaps
             url: "/indexing/sitemaps.html"
-          - name: Robots.txt files
+          - name: \/robots.txt Files
```
Comment (amanifarooque) [inline _config.yml:242]: This displays with both the forward and backslash on the Federalist preview. Does it error if you don't try to escape it with the backslash?

File: _data/hints.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ image_asset:
   logo_alt_text: "If you leave this blank, we'll use the site name provided on the Settings page as the default alt text."
 
 watcher:
-  name: "Enter a short, meaningful name - this will appear in the subject line of your alert."
+  name: "Enter a short, meaningful name; this will appear in the subject line of your alert."
```
Comment (amanifarooque) [inline _data/hints.yml:32]: @dawnpm Where does this appear on the site?

File: _posts/2012-11-16-november-2012.md [hunk 3]
```diff
@@ -20,13 +20,13 @@ To support you in collecting and reporting your site [search metrics]({{ site.ba
 1. **Top changing search terms (movement up/down).** Shows trending topics, what's hot or not. Use this report to investigate why people might be losing interest in once-popular content and consider archiving if it's no longer needed. For newly popular terms, create new content or update existing content to ensure it's current, accurate, and complete. 
 1. **Top clicked URLs for specific queries.** The most common results that searchers found most relevant or appealing, for specific search terms. Review these popular pages and other customer top tasks to make sure they are current, accurate, complete and compelling.
 
-### 2. Recommended Pages in Type-ahead Suggestions
+### 2. Recommended Pages in typeahead Suggestions
```
Comment (amanifarooque) [inline _posts/2012-11-16-november-2012.md:23]: Should this be title case?

File: _posts/2012-11-16-november-2012.md [hunk 4]
```diff
@@ -20,13 +20,13 @@ To support you in collecting and reporting your site [search metrics]({{ site.ba
...
 
 Searchers often know the exact page they're looking for. We've added a feature to help them find that page even more quickly. 
 
 As searchers type in their query, relevant [text Best Bets]({{ site.baseurl }}/admin-center/content/best-bets.html) are shown as Recommended Pages, as shown in the sample below for a search on [*fa&#8230;* on DOL.gov](https://search.usa.gov/search?query=jobs&amp;affiliate=u.s.departmentoflabor).
 
-![Type-ahead suggestions on DOL.gov's search results  page](https://d3qcdigd1fhos0.cloudfront.net/blog/img/tumblr_mdljlrljT71qid15q.png)
+![typeahead suggestions on DOL.gov's search results  page](https://d3qcdigd1fhos0.cloudfront.net/blog/img/tumblr_mdljlrljT71qid15q.png)
```
Comment (amanifarooque) [inline _posts/2012-11-16-november-2012.md:29]: Same comment here, should be title-case. I think we're deleting this anyway though with the other PR.

File: about/index.md [hunk 5]
```diff
@@ -6,29 +6,30 @@ last_modified_at: February 22, 2021
 sidenav: about_us
 ---
 
-Search.gov is a by-government, for-government web search engine. Available at no cost to federal agencies and the District of Columbia, we currently support the search boxes on about 2,200 websites, across 30% of federal domains. We are a product of Smarter IT Solutions, a division of the U.S. General Services Administration’s Technology Transformation Services, 
+Search.gov is a search engine built by the government, for the government. Available to federal agencies and the District of Columbia, we currently support over 300 million search queries a year for about 2,200 websites, across one third of federal domains.
```
Comment (amanifarooque) [inline about/index.md:9]: Is the District of Columbia still accurate?

File: about/policy/govt-urls.md [hunk 6]
```diff
@@ -27,32 +27,32 @@ Federal agencies [are required](https://www.whitehouse.gov/sites/whitehouse.gov/
...
-* Department of Defense websites for [recruiting](https://www.defense.gov/Resources/Military-Departments/DOD-Websites/category/Recruiting/) and [service academies](https://www.defense.gov/Resources/Military-Departments/DOD-Websites/category/Academy/)
-* [Travel and tourism](https://www.usa.gov/state-travel-and-tourism) websites for states and U.S. territories
-* [State lotteries](https://www.usa.gov/state-lotteries)
-* [Cooperative extensions](https://nifa.usda.gov/land-grant-colleges-and-universities-partner-website-directory?state=All&type=Extension)
-* [Combined federal campaigns](https://www.opm.gov/combined-federal-campaign/find-local-campaigns/#url=CFC-Zones)
-* [Government sponsored enterprises](https://en.wikipedia.org/wiki/Government-sponsored_enterprise#List_of_GSEs) (such as Fannie Mae)
-* Federal and state retirement systems
-* Task forces (such as the Preventive Services Task Force) and commissions (such as the 9/11 Commission)
-* A few select, non-goverment organizations (such as the Red Cross) and public-private partnerships
+* Federal, state, local, [tribal](https://www.bia.gov/tribal-leaders-directory), commonwealth, and territorial government agency websites.
```
Comment (amanifarooque) [inline about/policy/govt-urls.md:32]: Just for my knowledge, do all bullets need periods even if they're not complete sentences?

File: admin-center/dashboard/site-overview.md [hunk 7]
```diff
@@ -42,15 +42,15 @@ A list of the top 10 search terms with the greatest *gain* between yesterday and
...
 
 A list of the top 10 queries that returned no results.
 
 Use this data to help searchers find your content by adding a [Best Bet]({{ site.baseurl }}/admin-center/content/best-bets.html), updating your existing web pages, or both.
 
-For example, on one agency's website, a dozen searches for *frostline* returned no results. The frost line&mdash;also known as depth of frost or freezing&mdash;is the depth to which the groundwater in soil is expected to freeze. The agency's pages on the topic referred to this concept as *depth of freezing*. 
+For example, on one agency's website, a dozen searches for *frostline* returned no results. The frost line &mdash; also known as depth of frost or freezing &mdash; is the depth to which the groundwater in soil is expected to freeze. The agency's pages on the topic referred to this concept as *depth of freezing*. 
 
-## Top Queries with Low Click Thrus
+## Top Queries With Low Click Thrus
```
Comment (amanifarooque) [inline admin-center/dashboard/site-overview.md:53]: Were we dinged on consistency for "click through" vs. "click thru"?  I can't remember which we landed on

**Review by amanifarooque — CHANGES_REQUESTED** [review]
Left some comments, also can you remove the two .DS_Store that were included in this commit?

File: _data/hints.yml (hunk 2 above)
Comment (dawnpm) [reply _data/hints.yml:32]: These hints are the tooltip flags in the admin center. They get piped in.

File: _posts/2012-11-16-november-2012.md (hunk 3 above)
Comment (dawnpm) [reply _posts/2012-11-16-november-2012.md:23]: Yes, but this file was deleted with the other PR, so I didn't bother cleaning it up after the global edit.

File: _posts/2012-11-16-november-2012.md (hunk 4 above)
Comment (dawnpm) [reply _posts/2012-11-16-november-2012.md:29]: Yes, and this file was deleted with the other PR, so I didn't bother cleaning it up after the global edit.

File: about/policy/govt-urls.md (hunk 6 above)
Comment (dawnpm) [reply about/policy/govt-urls.md:32]: Yes, alas, that's AP style.

File: admin-center/dashboard/site-overview.md (hunk 7 above)
Comment (dawnpm) [reply admin-center/dashboard/site-overview.md:53]: No, but we should have been. I think we should go to "click through" and will make that edit throughout. Good catch.

File: about/index.md (hunk 5 above)
Comment (dawnpm) [reply about/index.md:9]: Yes, the 'available to' is the critical point here. We are still available to them.

**Comment by dawnpm** [comment]
@amanifarooque ready for re-review, thanks

**Comment by amanifarooque** [comment]
@dawnpm Good to merge!

**Review by amanifarooque — APPROVED** [review]

**Pull request merged**